app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -324,23 +278,59 @@
 
         </com.google.android.material.textfield.TextInputLayout>
 
+        <!-- Subregion -->
```

Comment:
I didn't build this yet, but wouldn't it make more sense for the subregion to appear before the zip?

Reply:
Oh yeah, it sure would. Good call